pkg/util/azureclient/azuresdk/armnetwork/usage.go

```diff
@@ -0,0 +1,27 @@
+package armnetwork
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v2"
```

Comment:
Because v2 was already being used when I started.
I don't object to using v6, but we have to replace all of them at once because we need to change clients first.
We can do it after we merge current PRs.